java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedServerSideMultipartManagerIT.java

```diff
@@ -65,13 +70,18 @@ public void beforeClass(@org.testng.annotations.Optional Boolean usingEncryption
         }
         mantaClient = new MantaClient(config);
 
+        // sanity check
+        if (!mantaClient.existsAndIsAccessible(config.getMantaHomeDirectory())) {
+            Assert.fail("Invalid credentials, cannot proceed with test suite");
+        }
+
         if (!mantaClient.existsAndIsAccessible(config.getMantaHomeDirectory()
                 + MantaClient.SEPARATOR + "uploads")) {
             throw new SkipException("Server side uploads aren't supported in this Manta version");
         }
 
         multipart = new EncryptedServerSideMultipartManager(this.mantaClient);
-        testPathPrefix = String.format("%s/stor/java-manta-integration-tests/%s",
+        testPathPrefix = String.format("%s/stor/java-manta-integration-tests/%s/",
```

Comment:
The missing slash meant files were created just outside the test directory with the test directory name as the first half of their filename. This is part of the reason test data wasn't being completely cleaned up after test runs.